include/private/target.h

```diff
@@ -96,4 +96,8 @@ COLD_FUNCTION
 int
 unsupported_target_is_open( const struct stumpless_target *target );
 
+static
+void
+close_unsupported_target( const struct stumpless_target *target );
+
```

Comment:
This declaration is unnecessary as the static `close_unsupported_target` function is only needed/used within the `src/target.c` file. Please remove it.